db/migrate/20220714124321_create_organizations.rb

```diff
@@ -0,0 +1,14 @@
+class CreateOrganizations < ActiveRecord::Migration[6.1]
+  def change
+    create_table :organizations do |t|
+      t.string :name, null: false
+      t.string :address
+      t.integer :phone
+      t.string :email, null: false
+      t.text :welcome_text, null: false
+      t.text :about_us_text
+
+      t.timestamps
+    end
```

Comment:
Podes agregar la migración de discard acá y nos ahorramos un archivo de migración:
```suggestion
  t.datetime :discarded_at
  t.index :discarded_at  
end
```

Reply:
Listo, modificado el archivo de migración de creación de la tabla organizations y eliminado el de añadir los campos para discard.